src/settingsdialogpaths.cpp

```diff
@@ -200,8 +200,8 @@ void PathsSettingsTab::on_browseGameDirBtn_clicked()
   QFileInfo oldGameExe(ui->managedGameDirEdit->text());
 
   QString temp =
-      QFileDialog::getOpenFileName(&dialog(), QObject::tr("Select game executable"),
-                                   oldGameExe.absolutePath(), oldGameExe.fileName());
+      QFileDialog::getOpenFileName(&dialog(), QObject::tr("Select game executable"), oldGameExe.absolutePath(),
+      oldGameExe.fileName().replace(QRegularExpression(" "), "?"));
```

Comment:
Hi, could you add a comment here to explain why this was needed? To avoid confusion for future contributors.

Reply:
Yes, I have added a comment. Let me know if it is clear enough, or needs changed.